UPGRADE-5.0.md

```diff
@@ -5,7 +5,8 @@ Security
 --------
 
  * The `ContextListener::setLogoutOnUserChange()` method has been removed.
-
+ * Using a `RouterInterface` that does not implement the `WarmableInterface` is not supported anymore.
```

Comment:
This needs to be documented under `FrameworkBundle` not `Security`. :)

Reply:
done